packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -0,0 +1,709 @@
+  render: () => {
+    return html`
+      <div class="flex flex-col gap-32">
+        <!-- sd-prose/sample-h1 -->
+        <div class="sd-prose">
+          <h1>H1 Nisi eu excepteur anim esse</h1>
+        </div>
+        <!-- sd-prose/sample-h2 -->
+        <div class="sd-prose">
+          <h2>H2 Nisi eu excepteur anim esse</h2>
+        </div>
+        <!-- sd-prose/sample-h3 -->
+        <div class="sd-prose">
+          <h3>H3 Nisi eu excepteur anim esse</h3>
+        </div>
+        <!-- sd-prose/sample-h4 -->
+        <div class="sd-prose">
+          <h4>H4 Nisi eu excepteur anim esse</h4>
+        </div>
+        <!-- sd-prose/sample-h5 -->
+        <div class="sd-prose">
+          <h5>H5 Nisi eu excepteur anim esse</h5>
+        </div>
+        <!-- sd-prose/sample-paragraph -->
+        <div class="sd-prose">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+        </div>
+        <!-- sd-prose/sample-paragraph-strong -->
+        <div class="sd-prose">
+          <p><strong>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</strong></p>
+        </div>
+        <!-- sd-prose/sample-inline-link -->
+        <div class="sd-prose">
+          <a href="#">Inline link</a>
+        </div>
+        <!-- sd-prose/sample-inline-link-strong -->
+        <div class="sd-prose">
+          <a href="#"><strong>Inline link</strong></a>
+        </div>
+        <!-- sd-prose/sample-list-unordered -->
+        <div class="sd-prose">
+          <ul>
+            <li>Unordered list level 1</li>
+            <li>Unordered list level 1</li>
+            <li>Unordered list level 1</li>
+          </ul>
+        </div>
+        <!-- sd-prose/sample-list-ordered -->
+        <div class="sd-prose">
+          <ol>
+            <li>Ordered list level 1</li>
+            <li>Ordered list level 1</li>
+            <li>Ordered list level 1</li>
+          </ol>
+        </div>
+        <!-- sd-prose/sample-list-ordered-nested -->
+        <div class="sd-prose">
+          <ol>
+            <li>
+              Ordered list level 1
+              <ol>
+                <li>Ordered list level 2</li>
+                <li>Ordered list level 2</li>
+              </ol>
+            </li>
+            <li>
+              Ordered list level 1
+              <ol>
+                <li>Ordered list level 2</li>
+                <li>Ordered list level 2</li>
+                <li>Ordered list level 2</li>
+              </ol>
+            </li>
+          </ol>
+        </div>
+        <!-- sd-prose/sample-media -->
+        <div class="sd-prose">
+          <figure>
+            <img src="./placeholders/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover" />
```

Comment:
Path has to be updated. cc @coraliefeil That's why the image is missing in Storybook.